### PR TITLE
Added note to API guide for filtering exact matches

### PR DESCRIPTION
**Pull request by tvo318**
##### SUMMARY
Addresses API behavior change  from previous release regarding default exact matches.

##### ISSUE TYPE
 - Bug, Docs Fix or other nominal change

##### COMPONENT NAME
 - Docs

##### AWX VERSION
latest

### CODE REVIEW

**Comment by tvo318** [comment]
@lucas-benedito can you take a look at this?

File: docs/docsite/rst/rest_api/filtering.rst [hunk 1]
```diff
@@ -122,3 +122,18 @@ Filtering based on the requesting user's level of access by query string paramet
 
 - ``role_level``: Level of role to filter on, such as ``admin_role``
 
+
+.. note::
+
+    Previous releases of AWX returned queries with **__exact** results by default, but you may find that the latest versions are returning a larger subset instead. As a workaround, set the ``limit`` to ``exact`` for the default filter. For example, ``/api/v2/jobs/?limitexact=lbenedit-test-node`` will result in:
```
Comment (lucas-benedito) [inline docs/docsite/rst/rest_api/filtering.rst]: the correct limit value should be `?limit__exact`. My original comment was broken by the automatic formatting.

Also, we should change the example from `lbenedit-test-node` to `example.domain.com`

**Review by lucas-benedito — APPROVED** [review]

File: docs/docsite/rst/rest_api/filtering.rst [hunk 2]
```diff
@@ -94,7 +94,7 @@ Field lookups may also be used for more advanced queries, by appending the looku
 
 The following field lookups are supported:
 
-- ``exact``: Exact match (default lookup if not specified).
+- ``exact``: Exact match (default lookup if not specified, see **Note** below).
```
Comment (oraNod) [inline docs/docsite/rst/rest_api/filtering.rst]:
```suggestion
- ``exact``: Exact match (default lookup if not specified, refer to the following note for more information).
```

File: docs/docsite/rst/rest_api/filtering.rst [hunk 3]
```diff
@@ -122,3 +122,18 @@ Filtering based on the requesting user's level of access by query string paramet
 
 - ``role_level``: Level of role to filter on, such as ``admin_role``
 
+
+.. note::
+
+    Previous releases of AWX returned queries with **__exact** results by default, but you may find that the latest versions are returning a larger subset instead. As a workaround, set the ``limit`` to ``?limit__exact`` for the default filter. For example, ``/api/v2/jobs/?limit__exact=example.domain.com`` will result in:
```
Comment (oraNod) [inline docs/docsite/rst/rest_api/filtering.rst]:
```suggestion
    Previous releases of AWX returned queries with **__exact** results by default, but you may find that the latest versions are returning a larger subset instead. As a workaround, set the ``limit`` to ``?limit__exact`` for the default filter. For example, ``/api/v2/jobs/?limit__exact=example.domain.com`` results in:
```

**Review by oraNod — APPROVED** [review]
LGTM but a couple grammar nits for you

**Review by oraNod — APPROVED** [review]
Thanks @tvo318

**Pull request merged**